client/server/ClassVisitor.js

```diff
@@ -36,17 +36,35 @@ class ClassVisitor {
                 if (!this.completionItems.find(function (elem, index, self) {
                     return elem.data === `id_${yidContext.text}_${currentClassId}`;
                 })) {
+                    const documentation = this.getDocumentation(node.field());
                     this.completionItems.push({
                         label: `${yidContext.text}`,
                         kind: vscode_languageserver_1.CompletionItemKind.Property,
                         data: `id_${yidContext.text}_${currentClassId}`,
-                        detail: `Attribute of class ${currentClassId}.`
-                        //,documentation: "Its documentation can come from predefinedObjects.xml"
+                        detail: `Attribute of class ${currentClassId}.`,
+                        documentation: `${documentation}`
                     });
                 }
             }
         }
     }
+    getDocumentation(options) {
+        try {
+            for (const option of options) {
+                if (option._optionName.text === "documentation") {
+                    const documentation = option._optionValues[0].text;
+                    if (documentation !== null && documentation !== undefined) {
+                        return documentation;
+                    }
+                }
+            }
+            return 'not documented';
+        }
+        catch (err) {
+            console.error(err);
+            return 'not documented';
```

Comment:
Duplicated raw string. :wink:
LOL I was tempted to suggest writing “undocumented”, but… https://www.wordnik.com/words/undocumented